generated/nifake/nifake.proto

```diff
@@ -379,11 +379,12 @@ message GetAnIviDanceWithATwistByteArrayResponse {
 }
 
 message GetAnIviDanceWithATwistStringRequest {
+  nidevice_grpc.Session vi = 1;
```

Comment:
The generated changes here are due to legitimate additional params and param name changes on a couple of functions between the old version of ni-fake metadata we were using (1.2.0d9) and the updated one (22.8.0d9999).

These are fine and expected as we update the driver version.